plat/common/arm/time.c

```diff
@@ -157,6 +158,8 @@ void ukplat_time_init(void)
 
 	/* Enable timer */
 	generic_timer_enable();
+	/* Initialize rtc */
+	_dtb_init_rtc(_libkvmplat_cfg.dtb);
```

Comment:
As the RTC is implemented as a driver that may or may not be available in a given platform, it should not be initialized here, but at the platform's initialization stage.

Reply:
This comment is unclear to me. How would moving the `_dtb_init_rtc()` call to the platform's initialization stage help?